src/blocks/scratch3_sensing.js

```diff
@@ -99,4 +100,11 @@ Scratch3SensingBlocks.prototype.getKeyPressed = function (args, util) {
     return util.ioQuery('keyboard', 'getKeyIsDown', args.KEY_OPTION);
 };
 
+Scratch3SensingBlocks.prototype.daysSince2000 = function()
+{
+    var start = new Date(2000, 1-1, 1); 
+    var today =   new Date(); 
+    return (today.valueOf() - start.valueOf()) / 1000 / 86400;
```

Comment:
Hey @CosmicWebServices,

Thanks for taking this on!

What about the timezone code we discussed in #268? It looks to me like JavaScript supports `getTimezoneOffset`, so we could adjust to UTC just like scratch-flash.